client/src/components/Message.js

```diff
@@ -24,11 +31,19 @@ const Message = ({ message: { user, text }, name }) => {
       </div>
     </div>
   ) : (
-    <div className="flex flex-col items-start p-2 rounded-t-xl rounded-r-xl w-max max-w-xs bg-green-500 shadow-md text-white m-2 overflow-hidden lg:max-w-lg">
-      <p className="pl-2 text-green-100">{user}</p>
-      <div>
-        <p className="pl-2">{text}</p>
+    <div class="grid grid-cols-6">
+      <div class="col-start-1 col-end-2 text-center">
+        <img class="inline object-cover w-8 h-8 rounded-full" src="https://p.kindpng.com/picc/s/78-785827_user-profile-avatar-login-account-male-user-icon.png" alt="Profile image"/><br/>
+        <div>
+          <p className="pr-2">{user}</p>
+        </div>
+      </div>
+      <div class="col-start-2 col-end-7">
+      <div className="flex flex-col items-start p-2 rounded-t-xl rounded-r-xl w-max max-w-xs bg-green-500 shadow-md text-white m-2 overflow-hidden lg:max-w-lg">
+          <p className="text-green-900 pr-2">{text}</p>
```

Comment:
Please, change the color of the text: 
```
<p className="pl-2 text-green-900">{user}</p>
```
For
```
<p className="pl-2 text-green-100">{user}</p>
```
Left messages has a darker background:
![Captura de pantalla 2021-10-06 a las 13 31 03](https://user-images.githubusercontent.com/38152841/136194482-817aad2e-3da0-479a-8426-2cd32c0425c4.png)